### PR TITLE
Add regular option to DRP and SDP

### PR DESCRIPTION
**Pull request by majapw**
In a follow-up, I'd like to make regular the default and set light as an option (which makes more sense to me)

to: @ljharb @gabergg @noratarano @erin-doyle

### CODE REVIEW

File: src/components/DateInput.jsx [hunk 1]
```diff
@@ -288,7 +292,6 @@ export default withStyles(({
   },
 
   DateInput_input: {
-    fontWeight: 200,
```
Comment (ljharb) [inline src/components/DateInput.jsx]: so this part is the breaking change?

Comment (majapw) [reply src/components/DateInput.jsx]: Yes precisely

**Comment by majapw** [comment]
@ljharb it was more that it seemed to me that a `regular` prop seems strange, whereas a light prop makes more sense when both are options (also matches our internal paradigms)

It's not strictly necessary to be breaking, it was just a preference

**Comment by ljharb** [comment]
I'd rather add a `regular` prop now, ship that, and then in the next semver-major, remove `regular` and make it the default.

**Comment by majapw** [comment]
@ljharb uh what?

**Comment by ljharb** [comment]
@majapw iow, `light` would have the same behavior with true or false; `light` and `regular` would be mutually exclusive; and then later, we'd remove "regular", and make `light` function as expected.

**Comment by majapw** [comment]
@ljharb That seems like the same amount of work as just switching them later on, it's just that the work is now instead of later.

**Comment by ljharb** [comment]
It's the same amount of work for *us*, but it'll be less work for developers to migrate to the next semver-major, since those using `light` won't have to do anything, and those using `regular` will merely have to remove the prop.

**Pull request merged**